docs/deployment.rst

```diff
@@ -73,14 +73,19 @@ latency access to both ZooKeeper and the Kafka brokers. Standard data center net
 JVM
 ~~~
 
-We recommend running JDK 1.7 u51, and using the G1 collector. If you do this (and we highly recommend it), make
+We recommend running the latest version of JDK 1.8 with the G1 collector (older freely available versions have disclosed security vulnerabilities).
```

Comment:
Ack. Had to remember how to toggle line length, different files across different projects are using completely different limits....